lib/get-selectors.js

```diff
@@ -8,6 +8,12 @@ function isNestedRule(node) {
   return /(?:at)?rule/.test(node.parent.type)
 }
 
+function isOnlyExtends(node) {
+  return node.nodes && node.nodes.length === 1 && node.every(function(child) {
+    return child.type === 'atrule' && child.name === 'extend'
+  });
+}
+
```

Comment:
That's a good point, I think that's a very real use case. I like the separation of having two functions, but I understand your point. If we kept the functions separate it would look like: 

```js
function isOnlyExtends(node) {
  return node.nodes && node.nodes.length && node.every(function(child) { 
    return child.type === 'atrule' && child.name === 'extend'
  });
}
```
What do you envision the repurposed version would look like?